templates/attribute_values.md.jinja2

```diff
@@ -16,4 +17,6 @@ Values of attribute {{ attribute_name }}
     |{{ value }}|
     {{- osm.tag_combinations(definition['osm_tags'])|default(' ', boolean=True) }}|
     {{- definition['description']|default(' ') }}|
-{% endfor -%}
+{% endfor %}
+
```

Comment:
Only to keep the output MarkDown code identical. We can remove them in a separate PR, as they shouldn't have any visual effect in the rendered documentation.